shell/platform/embedder/tests/embedder_a11y_unittests.cc

```diff
@@ -24,6 +24,21 @@ namespace testing {
 
 using EmbedderA11yTest = testing::EmbedderTest;
 
+TEST_F(EmbedderTest, CannotProvideNewAndLegacySemanticsCallback) {
+  EmbedderConfigBuilder builder(
+      GetEmbedderContext(EmbedderTestContextType::kSoftwareContext));
+  builder.SetSoftwareRendererConfig();
+  builder.GetProjectArgs().update_semantics_callback =
+      [](const FlutterSemanticsUpdate* update, void* user_data) {};
+  builder.GetProjectArgs().update_semantics_node_callback =
+      [](const FlutterSemanticsNode* update, void* user_data) {};
+  builder.GetProjectArgs().update_semantics_custom_action_callback =
+      [](const FlutterSemanticsCustomAction* update, void* user_data) {};
+  auto engine = builder.InitializeEngine();
+  ASSERT_FALSE(engine.is_valid());
+  engine.reset();
+}
+
 TEST_F(EmbedderA11yTest, A11yTreeIsConsistent) {
```

Comment:
This diff is a bit confusing. The contents of the old `A11TreeIsConsistent` was moved to a new test `A11yTreeIsConsistentUsingLegacyCallbacks`, which starts below on line 215. This `A11TreeIsConsistent` is new and tests the new embedder API.